app-template/src/types/portal-config.d.ts

```diff
@@ -56,12 +59,12 @@ type Markdown = {
 
 type StatefulButtonControl = {
   name: 'StatefulButtonControlWrapper'
-  props: StatefulButtonControlProps
+  props: any
```

Comment:
any way we can do better than 'any'?

Reply:
There was an odd issue with the typing that I was running into here, I'd like to leave it as is and I can add a TODO comment in another PR because I don't think it will be an easy bug to solve, thanks for pointing this out though.